parachain/Cargo.toml

```diff
@@ -67,7 +68,7 @@ polkadot-parachain = { git = "https://github.com/paritytech/polkadot.git", branc
 
 artemis-core = { path = "primitives/core" }
 artemis-runtime = { path = "runtime" }
-tarpaulin = "0.1.0"
+#tarpaulin = "0.1.0"
```

Comment:
Tarpaulin is giving me trouble, both on my machine while compiling, and on Github's CI